app/emails/templates/evidence_assessment_rejected.md

```diff
@@ -1,6 +1,6 @@
 Thank you for submitting a request for assessment in {{ domain_name }}.
 
-[View the decision]({{ frontend_address }}/login?next=%2F2%2Fseller-dashboard) and any feedback from the assessment team.
+[View the decision]({{ frontend_address }}/2/login?next=%2F2%2Fseller-dashboard) and any feedback from the assessment team.
```

Comment:
I don't think the next parameter will redirect using the newer login. Maybe we should leave as is until we can support that?

Reply:
removed